service/controller/app/v1/resource/chart/current_test.go

```diff
@@ -126,7 +125,7 @@ func TestResource_GetCurrentState(t *testing.T) {
 
 			var err error
 
-			var kc *kubeconfig.KubeConfig
+			var kc kubeconfig.Interface
```

Comment:
Fixing the signature. For the current state test we need to configure the kubeconfig library.